hax/hax/util.py

```diff
@@ -212,6 +212,8 @@ class ConsulUtil:
     def __init__(self, raw_client: Optional[Consul] = None):
         self.cns: Consul = raw_client or Consul()
         self.kv = ConsulKVBasic(cns=self.cns)
+        self.failed_svcs_lock = Lock()
+        self.failed_svcs: List[Fid] = []
```

Comment:
Simple list maybe enough for confds list only. But for all m0d processes it will make linear search in the list. Imagine large clusters with tens of thousands of processes - it will kill scalability and performance.

Reply:
Okay, let me try to just add confds here.

Reply:
Is this changed?

Reply:
Yes, in the updated patched we have removed this.